target/linux/ramips/image/mt7620.mk

```diff
@@ -696,6 +696,27 @@ define Device/netgear_ex6130
 endef
 TARGET_DEVICES += netgear_ex6130
 
+define Device/netgear_jwnr2010-v5
+  SOC := mt7620n
+  BLOCKSIZE := 4k
+  IMAGE_SIZE := 3776k
```

Comment:
0x3c0000 = 3840k ?

Reply:
I minus the 64k rootfs_data ? so, 3840k final ?